ensembl/htdocs/info/genome/variation/phenotype/sources_phenotype_documentation.html

```diff
@@ -412,7 +412,7 @@ <h2>List of sources providing phenotype/disease/trait associations for each spec
       <tr class="bg1">
         
         <td style="width:4px;padding:0px;margin:0px;border-top:1px solid #FFF;border-bottom:1px solid #FFF;background-color:#090"></td>
-        <td style="font-weight:bold"><a href="https://www.ebi.ac.uk/gene2phenotype" style="text-decoration:none" target="_blank">DDG2P</a></td>
```

Comment:
Our database still have the source as DDG2P. Should that cause confusion? Because in the website the phenotype is shown from source DDG2P - 
http://wp-np2-11.ebi.ac.uk:7070/Homo_sapiens/Gene/Phenotype?g=ENSG00000157227;r=14:22836560-22849041